surface/include/pcl/surface/ball_pivoting.h

```diff
@@ -0,0 +1,233 @@
+/*
+ * Software License Agreement (BSD License)
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2014-, Open Perception, Inc.
```

Comment:
This is a brand new file, created in 2017. You can not claim copyright for the time before it existed!